packages/fxa-auth-server/lib/routes/emails.js

```diff
@@ -890,8 +895,6 @@ module.exports = (
             }
           }
 
-          const account = await db.account(uid);
-
```

Comment:
Here's the bug in the tests. `account.emails` is "old" now since it's fetched first